tests/Avalonia.Benchmarks/packages.config

```diff
@@ -1,7 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file has been deleted from the filesystem but is still present in the project.